### PR TITLE
Syncano stripe-payments-cookbook

### PR DESCRIPTION
**Pull request by chinazor-allen**
#### What does this PR do?
Pin-points guidelines to integrating Syncano `stripe-payments` to an application

### CODE REVIEW

File: web/solutions/stripe-payments.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Stripe Payment Integration
+
+* Preparation: **2 minute**
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: s/minute/minutes

File: web/solutions/stripe-payments.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* American Express
+* American Express
+* Discover
+* Discover
+* Diners Club
+* JCB
+
+### Solution
+
+Our solution is establised using [stripe-payments](https://syncano.io/#/sockets/stripe-payments) to trigger payment service.
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: s/establised/established

File: web/solutions/stripe-payments.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Install syncano-client to interact with Syncano stripe-payments socket:
+
+```sh
+$ npm install syncano-client --save
+```
+
+or
+
+```HTML
+<script src="https://unpkg.com/syncano-client"></script>
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: `https://unpkg.com/@syncano/client`

File: web/solutions/stripe-payments.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+To use Syncano library, import:
+
+```javascript
+import Syncano from "syncano-client";
+```
+
+Create a connection by initializing Syncano object with instance name:
+
+```javascript
+const s = new Syncano("MY_INSTANCE_NAME");
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: const s = new SyncanoClient("MY_INSTANCE_NAME")

File: web/solutions/stripe-payments.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    return await s.post("stripe-payments/charge/charges", chargeParams);
+  } catch (error) {
+    return error.message;
+  }
+};
+```
+
+### Testing functionality
+
+Now you can create a payment form collecting `card details` from user ans pass to script file.
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: s/ans/and

File: web/solutions/stripe-payments.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+or
+
+```HTML
+<script src="https://unpkg.com/syncano-client"></script>
+```
+
+To use Syncano library, import:
+
+```javascript
+import Syncano from "syncano-client";
```
Comment (adamwardecki) [inline web/solutions/stripe-payments.md]: This should be better phrased here. 
`<script src="https://unpkg.com/syncano-client"></script>`
is the html version of 
`import Syncano from "syncano-client";`
which is used in es6 js apps.

we should make it clear these are two way's of achieving the same thing.

**Review by adamwardecki — CHANGES_REQUESTED** [review]

File: web/solutions/stripe-payments.md (hunk 1 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

File: web/solutions/stripe-payments.md (hunk 2 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

File: web/solutions/stripe-payments.md (hunk 3 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

File: web/solutions/stripe-payments.md (hunk 4 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

File: web/solutions/stripe-payments.md (hunk 5 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

File: web/solutions/stripe-payments.md (hunk 6 above)
Comment (chinazor-allen) [reply web/solutions/stripe-payments.md]: done

Comment (adamwardecki) [reply web/solutions/stripe-payments.md]: Not what I meant. 

1. When using webpack and es6 the way to handle the client lib is:
Shell:
```sh
npm i -S @syncano/client
```

Code:
```js
import SyncanoClient from "@syncano/client"

const s = new SyncanoClient("MY_INSTANCE_NAME");
```

2. And the vanilla js way:
```html
<script src="https://unpkg.com/@syncano/client"></script>
<script>
  const s = new SyncanoClient("MY_INSTANCE_NAME");
</script>
```

**Review by adamwardecki — CHANGES_REQUESTED** [review]

**Review by adamwardecki — APPROVED** [review]

**Pull request merged**